test/e2e/tests/test_helper.py

```
@@ -104,6 +134,18 @@ def _create_sa_token(sa_name, namespace=None, duration="10m"):
     return token
 
 
+def _delete_sa(sa_name, namespace=None):
+    """Delete a service account (best-effort, for cleanup)."""
+    namespace = namespace or _ns()
+    subprocess.run(["oc", "delete", "sa", sa_name, "-n", namespace, "--ignore-not-found"], capture_output=True, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)